opentelemetry-prometheus/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.10.0
+
+### Added
+
+- Added `prometheus-encoding` feature to export prometheus encoders #652 
+- Added `with_aggregator_selector` option #667
+
+### Changed
+
+- Update prometheus to 0.13 #644
```

Comment:
It should already be bumped in #644. Maybe you need to cherry-pick that commit or update your branch. According to the content in `opentelemetry-prometheus/Cargo.toml` below, this release doesn't ship #644.

Reply:
I think it does include #644, but #644 didn't update the changelog?

Reply:
Oh sorry about that. I misread :facepalm: Please just ignore me...